[Clang] Update of invoking SPIR-V Backend in Clang/Driver

VyacheslavLevytskyy opened the pull request:
We introduce here SPIR-V Backend support in Clang/Driver

VyacheslavLevytskyy commented:
It was agreed that we split https://github.com/intel/llvm/pull/16317 into three parts:

- CI
- Clang driver
- XFAIL & UNSUPPORTED in end-to-end tests.
This is the 2st part, copy-pasted from https://github.com/intel/llvm/pull/16317 as-is.

asudarsa commented:
Hi @VyacheslavLevytskyy 

The changes are small and look fine to me. However, the PR topic is a bit misleading. We have already introduced SPIR-V backend support and this change just updates the target triple. Can you please reword it accordingly? Or let me know if I might be missing something here.

Thanks

VyacheslavLevytskyy commented:
> Hi @VyacheslavLevytskyy
> 
> The changes are small and look fine to me. However, the PR topic is a bit misleading. We have already introduced SPIR-V backend support and this change just updates the target triple. Can you please reword it accordingly? Or let me know if I might be missing something here.
> 
> Thanks

Thank you, you are absolutely right. I changed the title.
